app/routes/events/components/EventDetail/index.tsx

```diff
@@ -333,7 +334,7 @@ export default class EventDetail extends Component<Props, State> {
               <Link to={groupLink}>{event.responsibleGroup.name}</Link>
             ) : (
               event.responsibleGroup.name
-            )}{' '}
+            )}
```

Comment:
I'm on mobile so might've missed it, but have you added styling to replace this space between the group name and the email?